index.html

```diff
@@ -10,7 +10,13 @@
   <link rel="stylesheet" href="application.css" type="text/css">
 </head>
 <body class="container">
-  <h2 class="text-big page-title">Front-End Tools</h2>
+  <div class="nav-container">
+    <div class="nav-bar">
```

Comment:
Try to use only one element instead this two `<div>`. You should use `<nav>` element instead